### DISCUSSION TITLE
Welcome to qelectrotech-elements Discussions!

### DISCUSSION THREAD

**Discussion by scorpio810** [qelectrotech/qelectrotech-elements#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DillmannFrench** [comment]
Great thanks for maintaining QelectroTech, I am an instrumentalist working in Nairobi, (Kenya) for [iLabAfrica](https://ilabafrica.strathmore.edu/) a research lab involved in the development of technical skills in this East African community.

**Comment by scorpio810** [comment]
You are Welcomme @DillmannFrench.

**Comment by Varung333** [comment]
Can someone help me out by providing A3 Templates and Title Blocks for A3 !
